[CAM-8384] - Document CAM CPC on vCF behavior (#2597)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Deploying Anywhere/orchestrating-vcloud-air-vcloud-director-deployments.md

{{{ "title": "Orchestrating vCloud Air and vCloud Director Deployments",
"date": "01-15-2019",
"author": "Julio Castanar",

> "keywords": ["cam","vcloud","vcloud air", "vcloud director", "add to vcloud", "deploy to vcloud", "resource names"],

Reviewer: ```suggestion
"keywords": ["cam", "cloud application manager", "vcloud", "vcloud air", "vcloud director", "provider"],
```

"attachments": [],
"contentIsHTML": false
}}}

**In this article:**

* [Overview](#overview)
* [Audience](#audience)
* [Prerequisites](#prerequisites)
* [Connect vCloud Air or vCloud Director in Cloud Application Manager](#connect-vcloud-air-or-vcloud-director-in-cloud-application-manager)
* [Deploy to vCloud Air or vCloud Director](#deploy-to-vcloud-air-or-vcloud-director)
* [Naming vCloud resources](#naming-vcloud-resources)
* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)

### Overview
Do you use vCloud Air as the VMware public cloud or vCloud Director as your VMware private cloud?  
Connect them in Cloud Application Manager for complete workload automation. Once you define the automation of your workloads in Cloud Application Manager, you can launch them to other clouds as well as VMware.  
vCloud Air and vCloud Director are available in Cloud Application Manager through the vCloud Director provider, which supports vCloud Director API version 5.1 and above.

### Audience

> Cloud Application Manager customers.

Reviewer: ```suggestion
Cloud Application Manager customers willing to use their vCloud Air or vCloud Director environments.
```

**In this article:**

> * [Pre-Requisites](#pre-requisites)

Reviewer: Please, add "Overview", "Audience" and prereqs

* [Connect vCloud Air or vCloud Director in Cloud Application Manager](#connect-vcloud-air-or-vcloud-director-in-cloud-application-manager)
* [Deploy to vCloud Air or vCloud Director](#deploy-to-vcloud-air-or-vcloud-director)
* [Naming vCloud resources](#naming-vcloud-resources)
* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)

> ## Pre-Requisites

Reviewer: Use three level titles at top level in the whole doc

* **Compute**

    | Option | Description |
    |--------|-------------|
    | Virtual CPUs | Specify the number of CPUs. See the [VMware docs](http://pubs.vmware.com/vca/index.jsp?topic=%2Fcom.vmware.vca.od.ug.doc%2FGUID-6180A0A5-E390-449D-8FD6-CD8601554EAE.html) for vCloud Air VM CPU limit. |
    | Memory | Specify the virtual RAM in GB. See the [VMware docs](http://pubs.vmware.com/vca/index.jsp?topic=%2Fcom.vmware.vca.od.ug.doc%2FGUID-6180A0A5-E390-449D-8FD6-CD8601554EAE.html) for vCloud Air VM RAM limit. |

* **Disks**

    | Option | Description |
    |--------|-------------|
    | Disk | Specify the storage capacity in GB. See the [VMware docs](http://pubs.vmware.com/vca/index.jsp?topic=%2Fcom.vmware.vca.od.ug.doc%2FGUID-6180A0A5-E390-449D-8FD6-CD8601554EAE.html) for vCloud Air VM storage limit. Adding disk storage capacity is supported. |

### Naming vCloud resources

When creating an instance, an automatic name is generated and used for VApps, VMs and Hostname, all with the same generated name. This way you can identify unregistered instances.

> This name generated has maximum length of 15 characters (max allowed in windows hostnames), and it's composed by this values:

Reviewer: ```suggestion
The generated name has a maximum length of 15 characters (max allowed in windows hostnames), and it's composed by these values:
```

```
InstanceName-ServiceId-InstanceNumber
```

> In the Policy Box it's possible specify you want more that one instance. An example is “jdedepl-i8oq0-1”.  

Reviewer: ```suggestion
In the Policy Box it is possible to specify more than one machine to be deployed, so the instanceNumber is added as a suffix in the name. An example is “jdedepl-i8oq0-1”.  
```

> Once created this name, **modifying the vApp name is not supported**, because it could affect to the instance operations. However, it's possible to change manually this name on VMs and Hostnames.  

Reviewer: ```suggestion
Once deployed, **modifying the vApp name is not supported**, because it could affect the instance operations through Cloud Application Manager. However, it's possible to change this name manually on VMs and Hostnames.
```

> In case want to rename a VM/Hostname, it would be needed to do the following steps:

Reviewer: ```suggestion
In case you want to rename a VM/Hostname, you need to perform the following steps:
```
